level1a/handlers/level1a.py

```diff
@@ -76,10 +76,10 @@
     "schedule_version": 0,
     "schedule_standard_altitude": 0,
     "schedule_yaw_correction": False,
-    "schedule_pointing_altitudes": [],
+    "schedule_pointing_altitudes": "[]",
```

Comment:
Maybe leave this as
```suggestion
    "schedule_pointing_altitudes": [],
```
 and ...

Reply:
Note: this will be reverted in #67